service/app/validators/numeric_literality_validator.rb

```diff
@@ -0,0 +1,7 @@
+class NumericLiteralityValidator < ActiveModel::EachValidator
+	def validate_each(record, attribute, value)
+		unless value =~ /\A[0-9]+\z/
+			record.errors[attribute] << (options[:message] || "is not numeric literal")
```

Comment:
__[rubocop]__
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)